src/sxastarter/src/lib/redirects.ts

```diff
@@ -0,0 +1,31 @@
+import { NextRequest, NextResponse } from 'next/server';
```

Comment:
Delete this file (looks like an accidental copy/paste of [src/sxastarter/src/lib/middleware/plugins/redirects.ts](https://github.com/sitecorelabs/sxa-starter/pull/20/files#diff-cc51be542ea0684ae519f851c5df6c711385453e9163f6030adbfee95c162fa0))